server.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * This file provided by Facebook is for non-commercial testing and evaluation purposes only.
+ * Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+var fs = require('fs');
+var path = require('path');
+var express = require('express');
+var bodyParser = require('body-parser');
```

Comment:
you might already know this, but generally for node projects, you want a package.json file in the root folder for local node_modules

http://stackoverflow.com/questions/12127114/how-to-write-a-package-json-file-so-that-all-dependencies-are-downloaded-with-n

that way, other people using your project can run 'npm install' and install all the libraries your project is dependent on